### ISSUE TITLE
Error when (r)ebuild or (l)oad the database 

### ISSUE THREAD

**Issue by Astrol99** [Jo-Dan/The-Machine#7]
So every time I either rebuild or load the database, an error pops up and says 'Failed to execute script machine' then exits out of terminal immediately and I managed to get a screenshot. Does anyone know the problem or fix? Thanks.

![screenshot 68](https://user-images.githubusercontent.com/41875791/44007167-7cb1712c-9e56-11e8-833f-6c3936f96202.png)

**Comment by Jo-Dan** [comment]
Unfortunately you'll need to build the opencv with extrasfrom source yourself to get it working.

**Comment by Astrol99** [comment]
Looks like I solved it in a different way, I was doing all of the methods I can think of to fix it until I used my last resort which was to uninstall and reinstall it again. Same with opencv, turns out, I had mutiple opencv instances such as opencv-python and opencv-python-headless. So what I did was pip uninstall them both. I also found out that I had the format wrong in subject.csv. What I did at first was Subject No.1,DAVID,ADMIN
Then I just erased the Subject No. part. They could be the origins of the problem. But thanks for the answer!
I also have a quick question, is there anything that I can do to improve the face_recognition software because it seems to identify me as ADMIN and UNKNOWN. It might be due to my poor camera quality. But thats all! Thanks!
